vscode-wpilib/resources/webviews/gradle2025import.html

```diff
@@ -2,111 +2,186 @@
+
+        <div class="wizard-navigation">
+          <div></div>
+          <!-- Empty div for alignment -->
+          <button id="next-to-step-2" class="vscode-button next-button" disabled>Next</button>
+        </div>
+      </div>
+
+      <!-- Step 2: Configure Project -->
+      <div class="wizard-step" id="step-2">
+        <div class="step-header">
+          <h2>Step 2: Configure Project</h2>
+          <p>Configure the new project's location and settings.</p>
+        </div>
+
+        <div class="project-row">
+          <div id="projectfolderdiv" class="project-label"><b>Base Folder</b></div>
+          <span>Select a base folder to place the new project into.</span>
+        </div>
+
+        <div class="project-row">
+          <input id="projectFolder" type="text" class="vscode-textfield" readonly />
+        </div>
+
+        <div class="project-row">
+          <button id="projectSelectButton" class="vscode-button">Select Destination Folder</button>
+        </div>
+
+        <div class="project-row">
+          <div class="project-label"><b>Project Name</b></div>
+          <div class="project-field-container">
+            <input id="projectName" type="text" class="vscode-textfield" />
+            <div id="projectNameError" class="project-error-text">Project name is required</div>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="project-label"><b>Team Number</b></div>
+          <div class="project-field-container">
+            <input
+              id="teamNumber"
+              type="text"
+              inputmode="numeric"
+              class="vscode-textfield"
+              placeholder="Optional"
+            />
+            <div id="teamNumberError" class="project-error-text">Invalid Team Number</div>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="vscode-checkbox">
+            <input id="newFolderCB" type="checkbox" checked />
+            <label for="newFolderCB">
+              <span class="icon">
+                <i class="codicon codicon-check icon-checked"></i>
+              </span>
+              <span class="text">Create a new folder</span>
+            </label>
+            <span class="checkbox-help">Creates a new folder at Base Folder/Project Name</span>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="vscode-checkbox">
+            <input id="desktopCB" type="checkbox" />
+            <label for="desktopCB">
+              <span class="icon">
+                <i class="codicon codicon-check icon-checked"></i>
+              </span>
+              <span class="text">Enable Desktop Support</span>
+            </label>
+            <span class="checkbox-help">Enables desktop simulation and unit testing</span>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <h3>Hardware Option</h3>
+          <p>Select a hardware option if applicable:</p>
```

Comment:
I think Hardware platform would be more clear then hardware option. And now that it's radio boxes and something needs to be chosen, if applicable can be deleted. I think it would also be preferable to list the default option first